persistent-scratch.el

```diff
@@ -154,7 +154,8 @@ representing the time of the last `persistent-scratch-new-backup' call."
           (let ((coding-system-for-write 'utf-8-unix))
             (write-region str nil tmp-file nil 0))
         (set-default-file-modes old-umask)))
-    (rename-file tmp-file actual-file t))
```

Comment:
This line is absolutely necessary for persistent-scratch to work: it relies on the write-new-then-rename idiom to atomically replace old saved state with new. (It's not a perfect solution because it loses inode attributes like permissions and xattrs, but that's an unix API deficiency that can't be fixed without introducing new APIs for something like atomic extent swap.) Without it, the new state is only saved to the temporary file, which is never read by anything.

Reply:
My bad ! Stupid copy / paste error from my init file ...
Should be fixed with new patch.